packages/docs/.env.example

```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_ALGOLIA_APP_ID=
+NEXT_PUBLIC_ALGOLIA_SEARCH_API_KEY=
+ALGOLIA_SEARCH_ADMIN_KEY=
```

Comment:
We can remove `ALGOLIA_SEARCH_ADMIN_KEY` from the example as it is used only on the CI

Reply:
✅